shared/index.cjsx

```diff
@@ -41,6 +41,7 @@ module.exports = {
 
   FreeResponse:           require './src/components/exercise/free-response'
   GetPositionMixin:       require './src/components/get-position-mixin'
+  Join:                   require './src/components/enroll/join'
   ScrollToMixin:          require './src/components/scroll-to-mixin'
```

Comment:
Hm adding `Abstract` doesn't seem to add anything to me that would help figure out what the thing does..  What I was thinking was that `Join` doesn't really say what is being joined.  When I think `join` I think SQL table, but it could be joining course, combining grades, etc.  `UserJoinCourse` might work, or just `JoinCourse`.  

/meh it's not really that big a deal and as long as you pick something we can easily grep for to rename it's probably good 😁